docs/cse/integrations/automation-service.md

```diff
@@ -0,0 +1,209 @@
+1. Click **Add to List** (or **Update** if editing an existing Automation).
+
+## Running Automations
+Automations will run automatically when an Insight is created or closed provided that:
+* The Automation is enabled, 
+* The Automation is configured to run on the trigger(s), and 
+* The Automation is an Insight Automation, or
+The Automation is an Entity Automation and the Insight contains one or more Entities of the Entity Type(s) configured in the Automation (this includes the primary and any related Entities).
+
+Automations can be run manually from the **Actions** drop-down on [Insight details](/docs/cse/records-signals-entities-insights/about-cse-insight-ui#insight-details-page) pages. (On [Entity details](/docs/cse/records-signals-entities-insights/view-manage-entities#about-the-entities-details-page) pages, Entity Automations can also be run manually from the **Automations** drop-down).
+
+<img src={useBaseUrl('img/cse/automations-actions-menu.png')} alt="Automations on the Actions menu" width="230"/>
+
+You will see three sections in the Action menu:
+* **Insight Automation**. Displays a list of all enabled Insight Automations configured to run manually.
+* **Entity Automation**. Displays a **Run Automations** option. Click **Run Automations** to open a dialog enabling you to select one or more Entity Automations to run (see below).
+* **Insight Actions**.  Displays a list of all valid legacy Insight Actions.
+
+:::tip
+You can run the same Automation more than once for a given Entity or Insight, but not at the same time. Additional attempts to run an Automation while an instance is running will result in an error.
+:::
+
+If you select **Entity Automation > Run Automations** you will be prompted to select one or more of the Entities included in the Insight:
+
+<img src={useBaseUrl('img/cse/automations-entity-menu.png')} alt="Entity Automation menu" width="600"/>
+
+1. Select one or more of the Entities listed or select **Select All Entities**. The selected Entities don’t have to be the same type. 
+1. Click **Next**.
+  A list displays of all Entity Automations that are enabled, configured to be run manually, and configured for at least one of the Entity Types you selected on the previous screen. 
+1. Select the Automations you wish to run and click **Run Automation**.
+  The Automations will run. The system will automatically run the appropriate Automations for the appropriate Entity Types.
+  <img src={useBaseUrl('img/cse/automations-entity-menu-2.png')} alt="Entity Automation menu with selections" width="600"/>
+  
+  In this example:
+    * The CarbonBlack Automation is configured for IP Addresses, Email Addresses, and Domain Names, so it will run four times (once for the Email Address and once for each IP Address selected on the previous screen).
+    * The nslookup Automation is configured to only run on IP Addresses so it will run three times.
+    * No Automation will run on the Hostname.
```

Comment:
```suggestion
In this example:
* The CarbonBlack Automation is configured for IP Addresses, Email Addresses, and Domain Names, so it will run four times (once for the Email Address and once for each IP Address selected on the previous screen).
* The nslookup Automation is configured to only run on IP Addresses so it will run three times.
* No Automation will run on the Hostname.
```